pyproj/_transformer.pyx

```diff
@@ -312,6 +319,7 @@ cdef PJ* proj_create_crs_to_crs(
         if not allow_ballpark:
             options[options_index] = b"ALLOW_BALLPARK=NO"
         options_index += 1
+    # TODO: add allow_superseded here??
```

Comment:
I didn't see anything about superseded transformers [in the PROJ docs](https://proj.org/development/reference/functions.html#c.proj_create_crs_to_crs_from_pj). Does that mean we can do without this option?

Reply:
If this is a feature you would like added, you can open an issue upstream to see what they say: https://github.com/osgeo/proj

Otherwise, you can pass on adding it since it currently cannot be changed.

Reply:
Great, I will remove this TODO comment. I'm a noob with Cython.